library/old_protocol/deserialize.cpp

```diff
@@ -112,4 +123,9 @@ int deserialize_lookup_new_response_body(dnet_node *n, const dnet_cmd &, data_po
 	return 0;
 }
 
+template
```

Comment:
At template instantiation I think we shouldn't write `template` at separate line, example: https://github.com/interiorem/elliptics/blob/4fc279c5a90efc4e2b2e68bd4c2d44448f28ea1f/bindings/cpp/async_result.cpp#L513